### ISSUE TITLE
eav table(s) aren't created (eav app not detected?)

### ISSUE THREAD

**Issue by therefromhere** [jazzband/django-eav2#33]
I'm not sure what I'm doing wrong, but the `eav_attribute` table isn't being created in my database when I run manage migrate.

Maybe I'm missing a setup step?

Steps to reproduce:

(with a fresh virtualenv, python3.6, on Ubuntu)

```
pip install django~=1.11.14 django-eav2~=0.11.0
django-admin startproject scratch
cd scratch
django-admin startapp foo
# create a simple models.py
cat << EOF > foo/models.py
from django.db import models
import eav

class Foo(models.Model):
    body = models.IntegerField()

eav.register(Foo)
EOF
```
```
# then enable contrib.sites (required?) and add eav and my `foo` app to scratch/settings.py :
INSTALLED_APPS = [
    'django.contrib.auth',
    'django.contrib.sites',
    'django.contrib.admin',
    'django.contrib.contenttypes',
    "eav",
    "foo",
]

SITE_ID = 1
```
and 

```
./manage.py makemigrations
./manage.py migrate
```
outputs:
```
Operations to perform:
  Apply all migrations: admin, auth, contenttypes, foo, sites
Running migrations:
  Applying contenttypes.0001_initial... OK
  Applying auth.0001_initial... OK
  Applying admin.0001_initial... OK
  Applying admin.0002_logentry_remove_auto_add... OK
  Applying contenttypes.0002_remove_content_type_name... OK
  Applying auth.0002_alter_permission_name_max_length... OK
  Applying auth.0003_alter_user_email_max_length... OK
  Applying auth.0004_alter_user_username_opts... OK
  Applying auth.0005_alter_user_last_login_null... OK
  Applying auth.0006_require_contenttypes_0002... OK
  Applying auth.0007_alter_validators_add_error_messages... OK
  Applying auth.0008_alter_user_username_max_length... OK
  Applying foo.0001_initial... OK
  Applying sites.0001_initial... OK
  Applying sites.0002_alter_domain_unique... OK
```

Note that manage migrate doesn't create an eav_attribute table, and therefore the following fails with "django.db.utils.OperationalError: no such table: eav_attribute"

```
from eav.models import Attribute
Attribute.objects.create(name='City', datatype=Attribute.TYPE_TEXT)
```

I fee like I'm doing something stupid, but I don't see what it is.

**Comment by IwoHerka** [comment]
Hi, thanks for the submission. You're doing everything fine. We had a bug in our `setup.py`. It didn't pick up the `migrations/` dir. The tests didn't pick it up either. Fixed it in `c182c51` and bumped the version on PyPI to `0.11.1`.

**Comment by therefromhere** [comment]
Thanks for the quick fix!
